README.md

```diff
@@ -1,56 +1,20 @@
-# Benefits Eligibility - Phase 3
+# usagov-bears-spa
 
-BEARS will be a proactive notification service that provides timely and personalized information about benefits eligibility, contextualized 
-by life events. Before we can do notifications and potentially proactive notifications, we need to be able to estimate eligibility and 
-figure out a way to get people to sign up and store their information. We also need to establish relationships with benefits programs so 
-we can keep eligibility information up to date. 
+## Build Setup
 
-### The following npm scripts are defined
+```bash
+# install dependencies
+$ npm install
 
- * `npm start`
-    * Runs the static site generator eleventy and webpack in watch mode.
- * `npm run build`
-    * Runs the production webpack build and eleventy build for the static site.
- * `npm run test`
-    * Runs the unit and functional tests.
+# serve with hot reload at localhost:3000
+$ npm run dev
```

Comment:
On first  install and run, I get a compile error because a file that gulp copies over wasn't found:
```
This relative module was not found:                                                                                                                                                                                                       
* ../assets/js/uswds in ./plugins/uswds.client.js
```

You'll probably need to add the gulp task to the beginning of the dev and start tasks for this to work.

Reply:
I added it to the tasks in a follow-up commit